src/main/java/com/iexec/sms/admin/AdminController.java

```diff
@@ -102,7 +102,7 @@ public ResponseEntity<Void> createBackup() {
      */
     @PostMapping("/{storageID}/replicate-backup")
     ResponseEntity<Void> replicateBackup(@PathVariable String storageID, @RequestParam String fileName) {
-        return performOperation(storageID, fileName, BackupAction.REPLICATE);
+        return performOperation(storageID, fileName, StringUtils.EMPTY, StringUtils.EMPTY, BackupAction.REPLICATE);
```

Comment:
Following the discussion on out of storage error, I think it would be more meaningful to write the call like this
```suggestion
        return performOperation(StringUtils.EMPTY, StringUtils.EMPTY, storageID, fileName, BackupAction.REPLICATE);
```